.github/workflows/ci.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ruby: ['2.0', '2.1', '2.2.2', '2.3.0', '2.4.0', '2.5.0', 'rbx']
```

Comment:
If opted to go this route we need to pick the latest bugfix revisions for each minor version here.